mms_message.go

```diff
@@ -0,0 +1,47 @@
+package messagebird
+
+import (
+    "net/url"
+    "time"
+)
+
+type MmsMessage struct {
+    Id                string
+    HRef              string
+    Direction         string
+    Originator        string
+    Body              string
+    Reference         string
+    Subject           string
+    MediaUrls         []string
+    ScheduledDatetime *time.Time
+    CreatedDatetime   *time.Time
+    Recipients        Recipients
+    Errors            []Error
+}
+
+type MmsMessageParams struct {
+    Subject           string
+    Reference         string
+    ScheduledDatetime time.Time
+}
+
+func paramsForMmsMessage(params *MmsMessageParams) (*url.Values, error) {
+    urlParams := &url.Values{}
+
+    if params == nil {
+        return urlParams, nil
+    }
+
```

Comment:
If new lines are `paragraphs` that help to separate code that has different purpose, is this blank line really needed?